### DISCUSSION TITLE
Is it possible to have two results csv file?

### DISCUSSION THREAD

**Discussion by steamededamame** [immersivecognition/unity-experiment-framework#151] [Q&A]
Hi, 

I've got a lot of things I'd like to log from my experiment and I'm wondering if it's possible to log them in different csv files so that the resulting file will be more readable. For example, I'd like to have one 'default' trail_results.csv file and another stimuli_positions.csv file. 

Thank you!

**Comment by jackbrookes** [comment]
You could do this manually by writing stimuli positions to a file using `session.SaveDataTable`. For this, you would have to build a UXFDataTable which contains your columns and rows. Alternatively, if you use one of the `trial.Save*` methods you can save data on each trial.

https://github.com/immersivecognition/unity-experiment-framework/wiki/Collect-Custom-Data

Personally, I think I would split them afterwords using a Python or R script or something, if it is useful to do so.

**Reply by steamededamame** [reply]
Thanks a lot for the suggestions! :)
